packages/editor/src/lib/editor/types/event-types.ts

```diff
@@ -16,6 +16,7 @@ export type TLPointerEventTarget =
 export type TLPointerEventName =
 	| 'pointer_down'
 	| 'pointer_move'
+	| 'long_press'
```

Comment:
naming question: because now we'd have:
- pointer
- click
- press

`long_press` does sound the most intuitive and i honestly don't have a better alternative ¯\\_(ツ)_/¯ 
`pointer_long_down`?? meehhh

Reply:
ehh long_press is pretty standard